### PR TITLE
OT248 56

### PR DESCRIPTION
**Pull request by maxif96**
• SlideResponseDTO was created
• SlideMapper was created
• SlideService and SlideServiceImpl was created
• Endpoint getById() was created
• Only ROLE_ADMIN is allowed to access to this endpoint
• Exception handled with message.properties
• It return a 404 NOT FOUND

### CODE REVIEW

**Comment by maxif96** [comment]
Error:

image

Success:

image

ROLE_USER (Unauthorized):

image

File: src/main/java/com/alkemy/ong/controller/SlideController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.persistence.EntityNotFoundException;
+
+@RestController
+@RequestMapping("/slides")
+public class SlideController {
+
+    @Autowired
+    SlideService slideService;
```
Comment (turcoamado) [inline src/main/java/com/alkemy/ong/controller/SlideController.java]: hace privada esta variable

File: src/main/resources/locale/messages.properties [hunk 2]
```diff
@@ -1,3 +1,4 @@
 error.name.notnull = Name can not be null.
 error.content.notnull = Content can not be null.
-error.image.notnull = Image can not be null.
\ No newline at end of file
+error.image.notnull = Image can not be null.
+notfound = with that id not found
```
Comment (turcoamado) [inline src/main/resources/locale/messages.properties]: te falto un "was" antes de "not", y deberias separarlo como los anteriores, con un punto -> `not.found = wit that id was not found`

**Review by turcoamado — CHANGES_REQUESTED** [review]
Te marque dos pavadas, esta muy bien

File: src/main/java/com/alkemy/ong/controller/SlideController.java (hunk 1 above)
Comment (maxif96) [reply src/main/java/com/alkemy/ong/controller/SlideController.java]: Cada vez son menos las que me olvido de ponerle el private jajaja

**Comment by maxif96** [comment]
Added changes requested

**Review by turcoamado — APPROVED** [review]

**Pull request merged**